start.php

```diff
@@ -15,7 +15,7 @@
 	 *
 	 * @var string
 	 */
-	$this_sdk_version = '2.9.0.4';
+	$this_sdk_version = '2.9.0.5';
```

Comment:
@DanieleAlessandra You will find in line 40 a note about theme. Let's add something for your changes too

<img width="1075" alt="CleanShot 2024-11-28 at 17 51 01@2x" src="https://github.com/user-attachments/assets/ca9aa1d1-6fba-4d85-ac7d-db31486b55e2">

Something like this:

```
	 * We updated the logic to support SDK loading from a subfolder of a theme as well as from a parent theme.
	 * This allows the SDK to be loaded from composer dependencies or from a custom `vendor/freemius` folder.
	 *
	 * @since 2.9.0.5
	 * @author Daniele Alessandra (@DanielleAlessandra)
```